elements/chart/src/main.js

```diff
@@ -54,7 +54,7 @@ export class EOxChart extends LitElement {
     /**
      * Data values passed on runtime. Requires a [named data source](https://vega.github.io/vega-lite/docs/data.html#named) in the provided `spec`
      *
-     * @type {{[dataSourceName: string]: import("vega-lite/types_unstable/data.js").InlineData}}
+     * @type {{[dataSourceName: string]: import("vega-lite/build/src/data").InlineData}}
```

Comment:
@A-Behairi I tried it with the new Vega version, and this now fails again: https://github.com/EOX-A/EOxElements/actions/runs/19530267147/job/55911350052?pr=1875#step:7:22